forch/utils.py

```diff
@@ -89,3 +90,12 @@ def dict_proto(message, proto_func, ignore_unknown_fields=False):
 def str_proto(message_text, proto_func):
     """Convert a string represented protobuf message to proto object"""
     return text_format.Parse(message_text, proto_func())
+
+
+def safe_mkdir(path):
+    try:
+        os.makedirs(path)
```

Comment:
How is this functionally different than

`os.makedirs(new_path, exist_ok=True)`

Reply:
ah, I didn't know there is such a parameter.. Updated